mne/io/base.py

```diff
@@ -2004,9 +2024,8 @@ def append(self, raws, preload=None):
         self.annotations = annotations
         for edge_samp in edge_samps:
             onset = _sync_onset(self, (edge_samp) / self.info['sfreq'], True)
-            self.annotations.append(onset, (1. / self.info['sfreq']),
-                                    'BAD boundary')
-
+            self.annotations.append(onset, 0., 'BAD boundary')
```

Comment:
'EDGE' is used for filtering, 'BAD' for epochs so we need both types

Because it has duration zero it shouldn't actually get excluded -- one of the tests I added that does epoching effectively verifies this